crates/oxc_linter/src/rules/oxc/only_used_in_recursion.rs

```diff
@@ -251,16 +463,62 @@ fn is_function_maybe_reassigned<'a>(
         })
 }
 
-// skipping whitespace, commas, finds the next character (exclusive)
-#[allow(clippy::cast_possible_truncation)]
-fn skip_to_next_char(s: &str, start: u32) -> u32 {
-    for (i, c) in s.char_indices().skip(start as usize) {
-        if !c.is_whitespace() && c != ',' {
-            return i as u32;
+fn get_member_ident<'a>(
+    mut expr: &'a JSXMemberExpression<'a>,
+) -> Option<&'a IdentifierReference<'a>> {
+    loop {
+        match &expr.object {
+            JSXMemberExpressionObject::IdentifierReference(ident) => return Some(ident),
+            JSXMemberExpressionObject::ThisExpression(_) => return None,
+            JSXMemberExpressionObject::MemberExpression(next_expr) => {
+                expr = next_expr;
+            }
         }
     }
+}
```

Comment:
use https://github.com/oxc-project/oxc/blob/1145341a92e433272b09786935cc4ffff177dbf0/crates/oxc_ast/src/ast_impl/jsx.rs#L48-L52 ?

Reply:
Resolved in: 5e4767a4c15157e3cc36cfe1c9789224c76c6b2c...7c4693327b264fb85b53c1fa437f9b29584b432d